app/frontend/courses/curricula/components/CoursesCurriculum__SubmissionsAndFeedback.res

```diff
@@ -245,6 +246,12 @@ let submissions = (
                   reaction->Reaction.reactionableId == submission->Submission.id
                 )
 
+              let showComments =
+                DomUtils.hasUrlParam(~key="comment_id") &&
+                DomUtils.hasUrlParam(~key="submission_id") &&
```

Comment:
You may as well omit `DomUtils.hasUrlParam(~key="submission_id")`.

I'm guessing this was added for performance, but I don't think it's worth the readability loss caused by the chaining of similar fragments in this long conditional.